deploy/deploy.sh

```diff
@@ -1,34 +1,20 @@
 #! /bin/bash
 
-# Modify THE WANDB_ENTITY value in `environment.sh` (line 16) before running this script.
+# Modify DH_DEPLOY_KEY value in `environment.sh` (line 19) before running this script.
 #
 # Deploys the crypto-real-time-inference application onto a Ubuntu or Debian-based
 # server.
 # 
 # Note that each script called is idempotent.
-# 
-# Args (positional)
-#   - 1) MODELS: selection value for "all", "nn", or "ml"
-
-
-# Validate passed in MODELS value
-if [ -n "$1" ]
-then
-  echo "Using $1 as MODELS selection 🟢"
-  MODELS=$1
-else
-echo "Using default 'all' as MODELS selection"
-  MODELS="all"
-fi
 
 # Setup the deployment environment
 bash deploy/setup_deploy_deb.sh
 
-# Set environment variables for model downloads
+# Set environment variables for deploy
 source ./deploy/environment.sh
 
-# Download models on disk
-poetry run python deploy/download_models_from_wandb.py -s $MODELS
+# Spin up the Prediction server
+bash deploy/deploy_container.sh
 
-# Spin up the prediction server
-poetry run python src/server.py
+# Spin up the Webhook server
```

Comment:
Once again, what might the consequences be if you run this file twice in a row?

Reply:
Added idempotency for webhook deployment:
- `deploy_webhook.sh`: the webhook will only be deployed if `curl -X POST` to the endpoint fails.